syntax ="proto2";

message Request
{
	required string command=1;
	required string pVersion=2 [default="Version: 1.0"];
	required string url=3;
	required string cId=4;
	required string cInfo=5 [default="Version: 1.0"];
	required string enconding=6 [default="utf-8"];
	required string content=7;
	required string signature=8;
}